UnitTests/Input/ResponderTests.cs

```diff
@@ -112,11 +113,11 @@ public void IsOverridden_True_IfOverridden ()
 		// MouseEvent is defined on Responder IS overriden on ScrollBarView (but not View)
 		Assert.True (Responder.IsOverridden (new ScrollBarView () { Text = "ScrollBarView overrides MouseEvent" }, "MouseEvent"));
 
-		//// OnKeyDown is defined on View
-		//Assert.True (Responder.IsOverridden (new View () { Text = "View overrides OnKeyDown" }, "OnKeyDown"));
+		// OnKeyDown is defined on View
```

Comment:
FWIW, this IsOverriden hack must be removed in v2. It should be removed as part of #https://github.com/gui-cs/Terminal.Gui/issues/3029

Reply:
Ok I leave as is and then per #3029 will be removed, right?